bundle/manifests/operator.antrea.vmware.com_antreainstalls.yaml

```diff
@@ -1,8 +1,8 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Should the `deploy/operator.antrea.vmware.com_antreainstalls_crd.yaml` be updated to be consistent with this file?

Reply:
It's a good idea indeed. I'l take care of it.

Reply:
@jwsui I noticed the operator repo has both the v1beta1 and v1 CR in https://github.com/vmware/antrea-operator-for-kubernetes/blob/main/deploy/. I think we might want to leave this PR as it is and push a new PR to remove the v1beta1 version for the CR.

Reply:
New Pr: https://github.com/vmware/antrea-operator-for-kubernetes/pull/29

Reply:
ok, removing redundant crd makes things easier.